analyzers/tests/SonarAnalyzer.Test/TestCases/SymbolicExecution/Roslyn/CalculationsShouldNotOverflow.CSharp11.cs

```diff
@@ -4,7 +4,7 @@ public class Sample
 {
     public void ListPattern(int[] array)
     {
-        if (array is [2147483600, ..])
-            _ = array[0] + 100; // FN
+        if (array is [2147483600 and var big, ..])
```

Comment:
Please keep the old one and add this as additional example

Reply:
The old one is not testing list pattern but array sensitivity. I can add it back, but it's not providing much value here.
Looking at it again, this file should not have been touched in the first place, though. I will revert the change.